backend/event/mutations/UpdateEpisodeOrderMutation.py

```diff
@@ -0,0 +1,39 @@
+from graphene import ID, Boolean, InputObjectType, Int, List, Mutation, NonNull, ResolveInfo
+from django.db import transaction
+
+from event.models import Episode
+from graphql_app.types.LettercraftErrorType import LettercraftErrorType
+
+
+class EpisodeOrderInput(InputObjectType):
+    id = ID(required=True)
+    rank = Int(required=True)
+
+
+class UpdateEpisodeOrderMutation(Mutation):
+    ok = Boolean(required=True)
+    errors = List(NonNull(LettercraftErrorType), required=True)
+
+    class Arguments:
+        episode_order_data = List(NonNull(EpisodeOrderInput), required=True)
+
+    @classmethod
+    def mutate(cls, root: None, info: ResolveInfo, episode_order_data: list[EpisodeOrderInput]):
+        # Map Episode IDs to new rank values
+        id_to_rank: dict[int, int] = {
+            int(episode.id): episode.rank for episode in episode_order_data
+        }
```

Comment:
I think these type declarations need to be `List` and `Dict` instead of `list` and `dict`, right?